person.rb

```diff
@@ -1,26 +1,33 @@
 require_relative 'nameable'
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
+  # @param [Integer] age
+  # @param [String] name
+  # @param [Boolean] parent_permission
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
+  # @return [String] name
   def correct_name
     @name
   end
 
+  # @return [Boolean]
   def can_use_services?
     of_age? && @parent_permission
   end
 
```

Comment:
- [ ] Kindly add the method for adding rentals that represents the `many-to-many` relation between `Book` and `Person` through `Rental` as requested by the previous reviewer.
```ruby
def add_rental(book, date)
  Rental.new(date, book, self)
end
```